internal/resend/src/client.tsx

```diff
@@ -101,4 +102,45 @@ export class Resend {
       console.error("Error occurred sending payment issue email ", JSON.stringify(error));
     }
   }
+
+  public async sendBatchBudgetExceeded(
```

Comment:
Adds the `batch` feature from Resend in order to send the budget spending limit emails.